public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderCorner.tsx

```diff
@@ -95,15 +96,16 @@ export class PanelHeaderCorner extends Component<Props> {
   }
 
   render() {
-    const { error } = this.props;
+    const { error, multipleErrors } = this.props;
     const infoMode: InfoMode | undefined = this.getInfoMode();
 
     if (!infoMode) {
       return null;
     }
 
-    if (infoMode === InfoMode.Error && error) {
-      return this.renderCornerType(infoMode, error, this.onClickError);
+    if (infoMode === InfoMode.Error && (error || multipleErrors)) {
+      const errorMsg = multipleErrors ? 'Multiple errors found. Click for more details' : error || '';
+      return this.renderCornerType(infoMode, errorMsg, this.onClickError);
```

Comment:
this is tricky because this is rendered as  `Tooltip` content and that has a delay of 150ms. Not even the `Tooltip` tests cover this but I can give it a try.